Templates/Per Tenant Extension/.github/workflows/PullRequestHandler.yaml

```diff
@@ -103,21 +98,14 @@ jobs:
       secrets: 'licenseFileUrl,insiderSasToken,keyVaultCertificateUrl,keyVaultCertificatePassword,keyVaultClientId,gitHubPackagesContext,applicationInsightsConnectionString'
       publishThisBuildArtifacts: ${{ needs.Initialization.outputs.workflowDepth > 1 }}
 
-  PostProcess:
+  StatusCheck:
     runs-on: [ windows-latest ]
     needs: [ Initialization, Build ]
-    if: (!cancelled())
+    if: always()
```

Comment:
What if the workflow run is canceled?

Then _StatusCheck_ would succeed, which isn't quite accurate.